android/app/src/androidTest/java/org/fossasia/openevent/DateTest.java

```diff
@@ -0,0 +1,362 @@
+package org.fossasia.openevent;
+
+import org.fossasia.openevent.utils.ISO8601Date;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Created by arpitdec5 on 02-06-2017.
+ */
+
+public class DateTest {
+
+    @Test
+    public void shouldConvertSingaporeDateStringToUTCDateString() {
```

Comment:
like for testing of getTimeZoneDateStringFromString(), i do following stuff:

1: use for date strings to check conversion from singapore time to utc
2: use two date strings to check conversion from singapore to amsterdam
3: use two date strings to check conversion from singapore to ist
4: use two date strings to check conversion from amsterdam to ist

you are saying to include all the above in one method right?